backend/api/migrations/0002_auto_20170310_2142.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2017-03-10 21:42
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import csv
+
+def load_initial_data(apps, schema_editor):
+    Business = apps.get_model("api", "Business")
+    Inspection = apps.get_model("api", "Inspection")
+
+    with open('../../../DOHMH_NYC_Restaurant_Inspection_Results.csv') as csv_file:
+        reader = csv.reader(csv_file, delimiter=',')
+        header = next(reader)
+        for row in reader:
+            Business.objects.bulk_create([
+                Business(camis=row[0], name=row[1],
+                         address=row[3] + ' ' + row[4] + ' ' + row[2] + ' ' + row[5],
+                         phone=row[6], cuisine_description=row[7])
+            ])
+            Inspection.objects.bulk_create([
+                    Inspection(business=row[0], record_date=row[16], inspection_date=row[8],
+                               inspection_type=row[17], action=row[9], violation_code=row[10],
+                               violation_description=row[11], critical_flag=row[12],
+                               score=row[13], grade=row[14], grade_date=row[15])
```

Comment:
I notice you're leaving these all your variables as string type, which is fine as long as all of the actual fields are defined as string fields in the models. I didn't check the models myself but it might be worth doing on your end if you're uncertain at all.

In any case, if there are problems with null values on non-nullable fields, or bad data types etc they will show up when you try and run this migration so you'll find out one way or another =)